task-maker-format/src/ioi/format/italian_yaml/mod.rs

```diff
@@ -403,6 +405,8 @@ struct TaskYAMLOrig {
     /// The number of decimal digits when displaying the scores.
     #[serde(default)]
     pub score_precision: usize,
+    /// The primary language used when importing the booklets.
```

Comment:
tasks?

Reply:
I was thinking it only affects how booklets are imported but maybe it's not used by cmsImportBooklet (or however it's called)? Maybe it's used only when using cmsImportTask, I'm not sure what's more appropriate.

Reply:
When would a booklet ever be imported?
It does affect import task.

Reply:
I looked up to cms source code and `cmsAddStatement` does not care about the italy_yaml format so `primary_language` is only used when importing tasks.